vcpkg/consume/binary-caching.md

```diff
@@ -0,0 +1,122 @@
+export`](../commands/export.md) for a different option to share built packages.
+
+See the [Binary Caching documentation](../users/binarycaching.md) to learn about the different [storage backends](../users/binarycaching.md#providers).
+
+## Set up a local binary cache
+
+### Default binary cache
+
+By default, vcpkg creates a local binary cache. This helps save time when installing the same
+package on the same machine.
+
+The location of the default binary cache depends on your operative system. 
+
+### [Windows](#tab/default-windows)
+
+Binary caching is enabled by default at the first valid location of:
+
+* The location specified by the `VCPKG_DEFAULT_BINARY_CACHE` environment variable
+* `%LOCALAPPDATA%\vcpkg\archives`
+* `%APPDATA%\vcpkg\archives`
+
+### [Non-Windows](#tab/default-non-windows)
+
+Binary caching is enabled by default at the first valid location of:
+
+* The location specified by the `VCPKG_DEFAULT_BINARY_CACHE` environment variable
+* `$XDG_CACHE_HOME/vcpkg/archives`
+* `$HOME/.cache/vcpkg/archives`
+
+---
+
+Use the `VCPKG_DEFAULT_BINARY_CACHE` environment variable to change the default binary cache
+location.
+
+The default binary cache uses a [`files` provider](../users/binarycaching.md#files-provider). This
+type of binary cache provider stores binary packages as compressed archives in a filesystem
+location.
+
+### Disabling the default binary cache
+
+To disable the default binary cache you need to set the [`VCPKG_BINARY_SOURCES` environment
+variable](../users/binarycaching.md#configuration-syntax) to `clear`.
+
+### [Windows](#tab/disable-windows)
+
+```PowerShell
+PS D:\vcpkg> $env:VCPKG_BINARY_SOURCES="clear"
+```
+
+### [Non-Windows](#tab/disable-non-windows)
+
+```bash
+$ VCPKG_BINARY_SOURCES=clear
+```
+
+---
+
+### Set up local binary cache locations
+
+You are not limited to the default binary cache location as your only local binary cache, it is
+possible to configure multiple filesystem locations as binary caches. You can even set up specific
+caches as read-only, write-only, or read and write.
+
+All binary caches are configured through the `VCPKG_BINARY_SOURCES` environment variable. The value
+of `VCPKG_BINARY_SOURCES` follows a  specific [configuration
+syntax](../users/binarycaching.md#configuration-syntax).
+
+
+### Example of multiple binary cache locations
+
+In this example we will set up two binary cache locations:
+
+* `\\remote\shared\vcpkg-cache` which is a network location, and
+* `D:\vcpkg-cache\archives`
+
+We want the remote location to be read-only, meaning we will use this location to restore packages
+but won't upload any locally-built binary packages to it. The second location will have both read
+and write permissions and we will use as a backup binary cache.
+
+Let's set the value of `VCPKG_BINARY_SOURCES` as follows `clear;files,\\remote\shared\vcpkg-cache,read;files,D:\vpckg-cache\archive`.
+
+Breaking down the configuration we have:
```

Comment:
"Breaking down the configuration we have:" - Consider rephrasing to "Breaking down this configuration:"